packages/synapse-react-client/src/components/AccessRequirementList/AccessRequirementList.stories.tsx

```diff
@@ -114,8 +114,8 @@ export const HasMetRequirements: Story = {
         ...getAccessRequirementHandlers(MOCK_REPO_ORIGIN),
         ...getAccessRequirementEntityBindingHandlers(MOCK_REPO_ORIGIN),
 
-        ...getAccessRequirementStatusHandlers(MOCK_REPO_ORIGIN, {
-          [mockManagedACTAccessRequirement.id.toString()]: {
+        ...getAccessRequirementStatusHandlers(MOCK_REPO_ORIGIN, [
```

Comment:
Changed `getAccessRequirementStatusHandlers` so the second argument is an array instead of a function. No logical changes in this file.